src/components/projects/ManuscriptPageContainer.tsx

```diff
@@ -170,6 +171,16 @@ const ManuscriptPageView: React.FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [state])
 
+  const trackAttrsPopper = useTrackAttrsPopper()
+
+  const onAppClick = useCallback(
```

Comment:
this looks redundant,  why not just pass "trackAttrsPopper" to "onClick"?

Reply:
I separated `trackAttrsPopper` so we could later add another callback to handle the case of figure and file inspector dropdown list